flutter/test/replay/scheduled_recorder_test.dart

```diff
@@ -38,7 +38,7 @@ class _Fixture {
     sut = ScheduledScreenshotRecorder(
       ScheduledScreenshotRecorderConfig(
         width: 1000,
-        height: 1000,
+        height: 750,
```

Comment:
The original test should work even with the changes:

```suggestion
        height: 1000,
```

Reply:
But, you previously wrote, that the `height` and `width` in the `RecorderConfig` is the `target height` and `target width`. Therefore the resulting image must be `1000x1000` regarding the config. But the test is looking for an image of the size of `1000x750`.

Reply:
Hmm, I guess I should have been clearer. These dimensions are width & height the replay recorder wants. That doesn't mean we should skew the image. Therefore, the only other option if we were to respect the dimensions precisely while keeping the actual screenshot with the right aspect ratio would be painting the image with the original aspect ratio over a blank image but that's not really necessary because just encoding the image of different dimensions to a video achieves the same. And because it's not necessary (and it would be slower), then there's no point in doing that. BTW that's also how ScreenshotQuality handled screenshot capture even before this PR, although it had only one "targetResolution()" for both width & height.

As for the test case. I believe it's a good test that the desired width/height are respected but the aspect ratio is kept, which we verify later in the test case. It's true that the test case with `height: 750` would also pass, it just wouldn't test that we respect the aspect ratio.

Reply:
> It's true that the test case with height: 750 would also pass, it just wouldn't test that we respect the aspect ratio.

Maybe worth adding an explicit test case if you think that would be clearer.